src/rollup/widgets/show-widget.js

```diff
@@ -78,15 +88,17 @@ export default function showWidget (w) {
         }
       }
     } else {
-      var hostNode = document.querySelector(widget.config.position);
+      var selector = widget.config.position; // legacy inline layout used position as selector.
+      if (widget.config.positionSelector != null) {
+        selector = widget.config.positionSelector;
+      }
```

Comment:
2¢: I favor using a ternary expression over reassignment

```suggestion
      var selector = widget.config.positionSelector == null
        ? widget.config.position // legacy inline layout used position as selector.
        :  widget.config.positionSelector; 
```